packages/fx-core/resource/package.nls.json

```diff
@@ -51,15 +51,12 @@
   "core.migrationToArmAndMultiEnv.ErrorMessage": "Upgrade Failed. Please check the error message in Output window to troubleshoot and fix. You can also click Learn More for the FAQ and how to upgrade manually.",
   "core.migrationToArmAndMultiEnv.PreCheckErrorMessage": "Upgrade Failed. '%s' doesn't exist or is not in json format. Please fix it and try again by running command (Teams: Upgrade project).\nYou can also click Learn More for the FAQ.",
   "core.consolidateLocalRemote.Message": "Your project will be upgraded to work with the latest version of Teams Toolkit. The upgrade will only backup and change the local config and manifest template file without changing your custom code. If you want to skip auto-upgrade, please install older version of Teams Toolkit (<=3.7.0)",
-  "core.consolidateLocalRemote.DifferentManifest": "Since Teams Toolkit will use manifest.remote.template.json as a single manifest template file after the upgrade and we detected you may have customized the manifest.local.template.json, you may need to update the manifest.template.json to include your change. See Learn More for details.",
+  "core.consolidateLocalRemoteWithAadManifest.Message": "Your project will be upgraded to work with the latest version of Teams Toolkit. The upgrade will only backup and change the local config, manifest template and projectSettings file without changing your custom code. If you want to skip auto-upgrade, please install older version of Teams Toolkit (<=3.7.0)",
```

Comment:
It is not specific to the file, seems not clear as previous upgrade message. Besides do we still need line 55 message in the new version? Since seems there is no scenario that only unify config will be upgraded right?

Reply:
> Besides do we still need line 55 message in the new version? Since seems there is no scenario that only unify config will be upgraded right?

We still need message on line 55, it is used by unify config migration when user changed teams app manifest file.  For project like SPFX without SSO support, it will only have unify config migration.

Reply:
updated